packages/jaeger-ui/src/model/ddg/visibility-codec.test.js

```diff
@@ -57,8 +57,8 @@ describe('visibility-codec', () => {
   });
 
   describe('encodeDistance', () => {
-    const ddgModel = transformDdgData([longSimplePath, simplePath], focalPayloadElem);
-    const shortModel = transformDdgData([shortPath], focalPayloadElem);
+    const ddgModel = transformDdgData([longSimplePath, simplePath].map(wrap), focalPayloadElem);
+    const shortModel = transformDdgData([shortPath].map(wrap), focalPayloadElem);
```

Comment:
Why not just wrap in the test resources file?

Reply:
the paths are used when validating as well, so the options are either:
* export wrapped paths, and need to unwrap in more places than it's used wrap
* export both unwrapped and wrapped paths, but that becomes more variables to keep track of
* export unwrapped paths, and wrap right before feeding to transform, which feels the cleanest due to the other uses of unwrapped paths